RMv4_Recipely/RMv4_Recipely/Favorites/FavoritesPresenter.swift

```diff
@@ -4,7 +4,12 @@
 import Foundation
 
 /// Интерфейс презентера модуля "Избранные рецепты"
-protocol FavoritesPresenterProtocol: AnyObject {}
+protocol FavoritesPresenterProtocol: AnyObject {
+    func checkIfFavouritesEmpty()
```

Comment:
```suggestion
    func checkIfFavoritesEmpty()
```
Favorites был написан с ошибкой, ниже также

Reply:
> Favorites был написан с ошибкой, ниже также

<img width="650" alt="image" src="https://github.com/poftnat/RMv4_Recipely/assets/149266332/c69e7169-7818-4998-8a80-8e2aff8eb220">
выбрала американский вариант